package.json

```diff
@@ -1,26 +1,5 @@
 {
-  "name": "vms-frontend",
```

Comment:
cmiiar`package.json` and `package-lock.json` should not change as i don't see dependency update on the new components. if above is the case please reset to previous version ( refer [dev commands](https://github.com/foodpanda-capstone-6/fe/blob/main/readme.md#development) for development)